### ISSUE TITLE
Download uploaded videos

### ISSUE THREAD

**Issue by hersche** [DanielnetoDotCom/YouPHPTube#259]
Hello again,

i find it usefull, when people easily could download movies from the internal network, for when they leave the network after. Also, you could easy surround the internal player and use other players direct (vlc for example).or also when a playstation 4 has decode-problems with the mp4 in it's webkit, etc

i did a change to have the link for me, but when you would implement it, you may could make this set-able to videos (to disallow for copyrighted videos or so) or let the admin decide if it's generally decide this (easier to implement, i think).

Also translation would be usefull for public.

Anyway, after line

https://github.com/DanielnetoDotCom/YouPHPTube/blob/master/view/include/video.php#L86

if've added

`   <?php echo "<a href='".$global['webSiteRootURL']."/videos/".$playNowVideo['filename'].".mp4'>Download ".$playNowVideo['title']." (rechtsklick auf link -> speichern als ...)</a>"; ?>`

What do you think about this?

best regards

vinz

**Comment by DanielnetoDotCom** [comment]
do you mean download the encoded videos on streamer?

**Comment by hersche** [comment]
yes. in my case, i use often the direct upload in the streamer code.

**Comment by DanielnetoDotCom** [comment]
Did you try to use the videos directory for external players?

**Comment by hersche** [comment]
ah, nope, didn't see this one. also usefull for my cases, but still not the same - my solution would be in context to the video. also, this is still a quick method for looking the stuff offline.

but if it's not sympathic to you, no problem for me to let it be as it is - i still know how to solve this for myself ;)

**Comment by hersche** [comment]
#563 : download-button is there, when activated via options.
